allauth/account/forms.py

```diff
@@ -371,7 +385,17 @@ def try_save(self, request):
         """Try and save the user. This can fail in case of a conflict on the
         email address, in that case we will send an "account already exists"
         email and return a standard "email verification sent" response.
+
+        If the honeypot was filled out then also send 'email verification sent'
+        response so that the bot thinks things worked.
         """
+        if honeypot_field_name := app_settings.SIGNUP_FORM_HONEYPOT_FIELD:
+            if self.cleaned_data[honeypot_field_name]:
+                user = None
+                adapter = get_adapter()
+                resp = adapter.respond_email_verification_sent(request, None)
+                return user, resp
```

Comment:
Open to discussion on exactly what to do when the honeypot is filled out. Best practices say to make it seem to the bot that 'everything worked' so this seemed like an easy response to copy.

Reply:
Given that there isn't any hard evidence of what works and what doesn't, I think it's best to not over complicate things and take the simplest approach possible.  So reusing the existing response seems fine.